src/main/webapp/style.css

```diff
@@ -1,8 +1,14 @@
 h1 {
-  font-family: Verdana, sans-serif; 
+  font-family: Verdana, sans-serif;
   font-size: x-large;
 }
 
-#emphasis {
+.emphasis,
+a {
   color: violet;
 }
+
+a.active,
+a:hover {
+  color: rebeccapurple !important;
+}
```

Comment:
nit: no newline

Reply:
Should have been more specific -- no newline at the end of the file.

Reply:
```suggestion
}

```